std/algorithm/comparison.d

```diff
@@ -546,6 +544,17 @@ elements.
 
 If the ranges are strings, $(D cmp) performs UTF decoding
 appropriately and compares the ranges one code point at a time.
+
+Params:
+    pred = The predicate used for comparison.
+    r1 = The first range.
+    r2 = The second range.
+
+Returns:
+    0 if both all ranges are entirely equal. -1 if $(D r1) is less than $(D r2)
```

Comment:
I think it's better to drop 'entirely' from 'if both ranges are entirely equal.'